packages/pwa/app/components/link/index.jsx

```diff
@@ -11,23 +11,43 @@ import {Link as SPALink, NavLink as NavSPALink} from 'react-router-dom'
 import {buildPathWithUrlConfig} from '../../utils/url'
 import useSite from '../../hooks/use-site'
 import useLocale from '../../hooks/use-locale'
+import {matchRoute} from '../../utils/routes-utils'
+import routes from '../../routes'
 
 const Link = React.forwardRef(({href, to, useNavLink = false, ...props}, ref) => {
     const _href = to || href
     const site = useSite()
     const locale = useLocale()
 
+    //TODO: Add configuration value to the default.js config defining the SFRA hostname
+    const urlHostname = 'https://sfra-site.com'
```

Comment:
When thinking about naming for this, remember that it could also be a Site Genesis site or something else. We might want a generic-y setting.